frontend/src/scenes/session-recordings/playlist/sessionRecordingsPlaylistLogic.ts

```diff
@@ -543,6 +552,13 @@ export const sessionRecordingsPlaylistLogic = kea<sessionRecordingsPlaylistLogic
     selectors({
         logicProps: [() => [(_, props) => props], (props): SessionRecordingPlaylistLogicProps => props],
 
+        listAPIAsQuery: [
+            (s) => [s.featureFlags],
+            (featureFlags) => {
+                return !!featureFlags[FEATURE_FLAGS.REPLAY_LIST_RECORDINGS_AS_QUERY]
```

Comment:
One thing that always catches me about the query endpoint is the caching. You might need to disable it so that the list refreshes (or maybe the caching is desirable)

Reply:
yep, i'm slightly wary of moving to the query endpoint tbh 🙈 
(just from a "i don't know how this works right now" viewpoint